lib/datadog/tracing/contrib/utils/quantization/http.rb

```diff
@@ -59,22 +59,24 @@ def query(query, options = {})
 
             def query!(query, options = {})
```

Comment:
This method now has crazy logic to implement the expected behaviour, especially when `:show` (whether `Array` or `:all`), `:exclude`, and `:obfuscate` interact.